src/codewhisperer/service/transformByQHandler.ts

```diff
@@ -84,13 +84,33 @@ export async function getOpenProjects() {
  */
 export async function validateProjectSelection(project: vscode.QuickPickItem) {
     const projectPath = project.description
+    const isMaven = await checkIfMaven(projectPath!)
```

Comment:
Would it make more sense to have a single method that returns the build type (Maven, Gradle, and Unknown)? It seems strange that `checkIfMaven` returns `true/false` and `checkIfGradle` returns a string (which should probably be an enum anyway...)

Reply:
Yep, agreed, fixing now